os/include/debug.h

```diff
@@ -499,6 +499,13 @@ int get_errno(void);
 #define mllvdbg(...)
 #endif
 
+#if defined(CONFIG_MM_ASSERT_ON_FAIL) && defined(CONFIG_APP_BINARY_SEPARATION) && defined (__KERNEL__)
+#define mfdbg lldbg
+#else
+#define mfdbg mdbg
+#endif
```

Comment:
Could you check format? It's different with above and below. I think you need to add this near line 1311.

Reply:
Moved.

Reply:
You need to keep this here as well with proper format. (also need to add this at line 1311.)

Reply:
Yes Done.